<hr>

# Module 79.1 - Performance Appraisal and Return Calculations

<hr>

### LOS 79.a: Describe the performance appraisal of alternative investments.

Alternative investments are typically exposed to greater risks than unleveraged long-only traditional investments. These additional risks arise from the following:

* Timing of cash flows over an investment's life cycle
* Use of leverage by fund managers
* Valuation of investments that may or may not have observable market prices
* Complexity of fees, taxes, and accounting

Ideally, returns on alternative investments should be adjusted for these risks, although that may be difficult in practice. It is clear, however, that evaluating alternative investment returns (or expected returns) without considering these additional risks would be naïve and possibly misleading.

### Timing of Cash Flows

Alternative investments often have a life cycle that exhibits three phases:

1. In the **capital commitment phase**, a fund's managers are identifying investments and making **capital calls** from the partners. Recall that limited partners commit a stated amount of capital that they will invest, but they do not necessarily deliver the entire amount immediately in cash. Managers make capital calls as they identify investments for which they require cash. Because of these cash outflows and the long-term nature of the typical investments, returns tend to be negative during the capital commitment phase.

2. During the **capital deployment phase**, the managers fund, and often involve themselves directly in, the firms or projects in which they invest. Returns typically remain negative in this phase, especially if the investments are in start-up companies or troubled firms that the managers are attempting to turn around.

3. If the fund's investments succeed and begin to generate income and cash flows, the fund enters a **capital distribution phase** during which its returns turn positive and accelerate.


A **J-curve effect** (illustrated in  **J-Curve Example**) reflects the norm of negative returns in the capital commitment phase, followed by increasing returns in the capital deployment phase and maximum returns in the capital distribution phase. Returns may reach a plateau toward the end of a fund's life as the managers exit any remaining investments.


*J-Curve Example*


# img src="">

Given the variability of cash flows over a fund's life and the importance of management decisions in the timing and magnitude of after-tax cash flows, an IRR over the life of a fund is the most appropriate measure of after-tax investment performance. (Recall from Quantitative Methods that a money-weighted rate of return, which is an IRR, is appropriate when a manager controls the timing of cash inflows and outflows.) A drawback to using IRRs is that they rest on assumptions about the cost of capital for cash outflows and the reinvestment rate for cash inflows.


A simpler measure of investment success is the **multiple of invested capital** (or **money multiple**)—the ratio of total capital returned plus the value of any remaining assets, to the total capital paid in over the life of the investment. Because this measure does not consider the timing of cash inflows and outflows, which can affect annual returns on invested capital significantly, it can be considered somewhat naïve.

### Use of Leverage

Some alternative investments, particularly hedge funds, use borrowing to magnify their gains (at the risk of magnifying losses). Hedge funds may arrange margin financing with **prime brokers** or employ leverage by means of derivatives.

To state the effect of leverage on returns, consider a fund that can invest the amount V0 without leverage, and earn the rate of return r. The fund's unleveraged portfolio return (as a money amount) is simply this: r × V0. Now, let's say this fund can borrow the amount VB at an interest rate of rB, and earn r by investing the proceeds. The fund's leveraged portfolio return (again, as a money amount), after subtracting the interest cost, then becomes:
\
\
$\qquad r \times (V_0 + V_B)-(r_B \times V_B)$
\
\
Thus, stated as a rate of return on the initial portfolio value of $V_0$, the leveraged rate of return is as follows:
\
\
$\qquad \text{leveraged return } = \large\frac{r(V_0 + V_B)-r_B V_B}{V_0}$
\
\
One of the reasons that funds use leverage is that some strategies attempt to exploit relatively small pricing anomalies that might not produce meaningful results without leverage. A risk from using leverage is that a lender may issue margin calls if a fund's equity position decreases below a certain level. These can result in a fund having to realize losses by closing positions or liquidating investments at unfavorable prices. If the fund must sell a large position in a security, doing so may depress its price further. Another important risk of funds that depends on leverage is that lenders may limit their access to additional borrowing.


### Valuation of Investments

As we have discussed, many alternative investments involve illiquid assets that do not trade frequently in transparent markets. While funds must recognize investments at fair value to comply with accounting standards, fair value might rest on assumptions of which an investor in a fund should be aware. **A fair value hierarchy** groups these assumptions into the following three levels:

* *Level 1*. The assets trade in active markets and have quoted prices readily available, such as exchange-traded securities.

* *Level 2*. The assets do not have readily available quoted prices, but they can be valued based on directly or indirectly observable inputs, such as many derivatives that can be priced using models.

* *Level 3*. The assets require unobservable inputs to establish a fair value, such as real estate or private equity investments, for which there have been few or no market transactions.

Particularly for Level 3 investments, the absence of market activity can result in valuations that remain near their initial cost for long periods. As a result, these values might not reflect the actual exit costs of the investments. Importantly, this relative lack of change in fair values can make reported returns for alternative investments appear higher, less risky, and less correlated with traditional investments than they really are.



### LOS 79.b: Calculate and interpret alternative investment returns both before and after fees.

**

We have seen how margin calls may require a leveraged fund to exit investments at unfavorable prices and unintended times. A similar risk arises from investor **redemptions**. The more negative a fund's returns, the more likely investors are to ask the manager to redeem their positions. For this reason, and because of the J-curve effect of negative returns in the early years, alternative investment funds (particularly hedge funds) typically take measures to restrict early redemptions.

A **lockup period** is the time after initial investment over which limited partners either cannot request redemptions or incur significant fees for redemptions. A **notice period** (typically between 30 and 90 days) is the amount of time a fund has to fulfill a redemption request. Notice periods allow time for managers to reduce positions in an orderly manner. Fund managers often incur significant transactions costs when they redeem shares. **Redemption fees** can offset these costs. Managers may also have the discretion to implement a **gate** that restricts redemptions for a temporary period.

Investors should be aware that other investors may receive terms that differ from those stated in the partnership agreement. Customized fee structures are contained in **side letters** with individual investors detailing how their terms differ from those in the standard offering documents.

Although "2 and 20" and "1 and 10" were, at one time, fairly standard fee structures for fund and fund-of-funds investments, these fee structures continue to be under competitive pressure. Investors making larger commitments can negotiate lower fees. There can also be a tradeoff between liquidity provisions and fees. Investors can negotiate for lower fees or better liquidity (shorter lockups and notice periods). Hurdle rates, hard versus soft hurdles, and catch-up provisions may also be subject to negotiation.

Early investors in a fund may also receive lower fees or better liquidity terms as an incentive to invest at the fund's inception. The investment interests of early investors who receive such relatively better terms are called **founders class shares**.

Annual investor fees can also be **either-or fees**, the maximum of the management fee or the incentive fee. Under such a structure, with a 1\% management fee and a 30\% incentive fee, investor fees each year would simply be the management fee unless the calculated incentive fee is higher. Such a structure may also stipulate that the 1% management fee be subtracted from the incentive fee in a subsequent year.

### Biases in Alternative Investment Returns

Numerous alternative investment indexes exist to measure historical performance. However, they may not provide much meaningful information on the asset class because each fund's structure is unique, and the funds that exist at any given time can be in widely different phases of their life cycles. One way around this latter issue is to compare funds that originated in the same **vintage year**.

The effect of **survivorship bias** is greater for a hedge fund database than for other asset classes because by some estimates, more than 25% of hedge funds fail in the first three years of their existence. An index that does not include failed funds will overstate the returns and understate the risk of hedge funds as an asset class. These effects may be magnified by **backfill bias**, which occurs when managers only select their successful funds for inclusion in indexes.




### Return Calculations for Alternative Investments

Before-fee returns on alternative investments are calculated the same way we calculate returns on any investment. The calculation of holding period returns and periodic rates of return, both with and without interim cash distributions, is described in Quantitative Methods and expanded on by application in other topic areas.

Calculating after-fee returns simply requires adjustment of the cash flows or values for the various fees involved, typically management and performance fees. For a simple case in which management fees are a fixed percentage of end-of-period assets and performance fees are a fixed percentage of total return with no hurdle rate, we can state the general partner's total fees in money terms as follows:

$\qquad  \text{total fees} = mV_1 + \max \big[0, p(V_1-V_0)\big]$

where:

$m$ = management fee

$p$ = performance fee

$V_0$ = beginning-of-period assets

$V_1$ = end-of-period assets

<p> <br> </p>

This is the rate of return for an investor after fees:

$\qquad r = \Large\frac{V_1 - V_0  - \text{ total fees}}{V_0}$

Fee structure provisions, such as hurdle rates and high-water marks, make calculating total fees more complex than this simple formula. Other provisions may state whether the performance fee is net of the management fee or independent of it, or whether the management fee is based on beginning-of-period assets or end-of-period assets.

>**Professor's Note**
\
Because none of these provisions are standard across alternative investment funds, exam questions that require calculations will have to specify all of those that apply. Do not assume any provision applies unless it is stated in the question.

Some examples will illustrate the application of various fees and the relevant terminology.

<hr>

### Example: Hedge fund fees
BJI Funds is a hedge fund with a value of €110 million at initiation. BJI Funds charges a 2\% management fee based on assets under management at the beginning of the year and a 20\% performance fee with a 5\% soft hurdle rate, and uses a high-water mark. Performance fees are calculated on gains net of management fees. The year-end values before fees are as follows:

* Year 1: €100.2 million
* Year 2: €119.0 million

Calculate the total fees and the investor's after-fee return for both years.



In [52]:
#Example Hedge fund fees

# Year 1 
# --------------------------------
print("----------------------")
print("Year 1")
print("----------------------")


# Calculate management fee yr 1
mgmt1 =  110000000 * 0.02
print("Management fee = ",'{:,}'.format(round(mgmt1,2)),"euros")


# Calculate Return net of management fees:
rnomf1 = (((100.2-2.2)/110) -1) * 100
print("Return net of management fees:",'{:,}'.format(round(rnomf1,2)), "%")
print("\n")
print("There is no performance fee because the return after") 
print("the management fee is less than the 5% hurdle rate.")
print("\n")

# Calculate total fees of year 1
feesyr1 = (((100200000 - 2200000)/110000000) - 1) * 100
print("total after fees return Year 1:",'{:,}'.format(round(feesyr1,2)), "%")


# Year 2
# --------------------------------
print("\n")
print("----------------------")
print("Year 2")
print("----------------------")

# Calculate management fee yr 1
mgmt2 =  98000000 * 0.02
print("Management fee for the year 2      = ",'{:,}'.format(round(mgmt2,2)),"euros")

# Year-end value net of management fee:	
YEmgmt2 = 119000000 - mgmt2
print("Year-end value net of management fee:",'{:,}'.format(round(YEmgmt2,2)),"euros")
print("\n")
print("The high-water mark is $110 million.")

highwater2 = ((117040000 - 110000000)/110000000) *100
print("Year 2 return net of management fee, above high-water mark:",'{:,}'.format(round(highwater2,2)),"%")

print("\n")
print("----------------------")
print("Performance fees")
print("----------------------")

# Calculate total fee
total_fee = ((117040000 - 110000000) * .20)  + mgmt2
print("Total fees of years 1 and 2: ",'{:,}'.format(round(total_fee,2)),"euros")


# Calculate Year 2 year-end value after fees:
EOY2_fees = 119000000 - total_fee
print("Year 2 year-end value after fees:",'{:,}'.format(round(EOY2_fees,2)),"euros")

EOY2_feepct = ((115.63/98) - 1) *100
print("Year 2 year-end value after fees:",'{:,}'.format(round(EOY2_feepct,1)),"%")

----------------------
Year 1
----------------------
Management fee =  2,200,000.0 euros
Return net of management fees: -10.91 %


There is no performance fee because the return after
the management fee is less than the 5% hurdle rate.


total after fees return Year 1: -10.91 %


----------------------
Year 2
----------------------
Management fee for the year 2      =  1,960,000.0 euros
Year-end value net of management fee: 117,040,000.0 euros


The high-water mark is $110 million.
Year 2 return net of management fee, above high-water mark: 6.4 %


----------------------
Performance fees
----------------------
Total fees of years 1 and 2:  3,368,000.0 euros
Year 2 year-end value after fees: 115,632,000.0 euros
Year 2 year-end value after fees: 18.0 %


**Answer:**
\
\
Year 1:
\
\
Management fee:	
\
\
110.0 million × 2\% = €2.2 million	
\
\
Return net of management fees:	
\
\
$\Large\frac{100.2−2.2}{110.0} \small –1 = –10.9\%$	
\
\
There is no performance fee because the return after the management fee is less than the 5\% hurdle rate.

<p> <br> </p>


Total fees:
\
\
€2.2 million	
\
\
Ending value net of fees:	
\
\
100.2 million – €2.2 million = €98.0 million	
\
\
Year 1 after-fees return:	
\
\
$\Large \frac{98.0}{110.0} \small –1  = –10.9\%$

<p> <br> </p>

Year 2:
\
\
Management fee:	
\
\
98.0 million × 2\% = €1.96 million	
\
\
Year-end value net of management fee:	
\
\
€119.0 – €1.96 = €117.04 million	
\
\
The high-water mark is €110 million.
\
\
\
Year 2 value net of management fee, above high-water mark:
\
\
117.04 million – 110.0 million = €7.04 million
\
\
Year 2 return net of management fee, above high-water mark:	
\
\
$\Large\frac{7.04}{110.0} \small = 6.4\%$	

<p> <br> </p>

Due to the high-water mark, the performance fee is calculated based on gains in value above €110 million.

The performance fee is calculated on the entire gain above the high-water mark because 6.4\% is greater than the soft hurdle rate. If the 5\% was a hard hurdle rate, the performance fee would be calculated only on the gains more than 5\% above the high-water mark.

>Performance fee:	
\
\
7.04 × 0.20 = €1.41 million	
\
\
Total fees:	
\
\
€1.96 million + €1.41 million = €3.37 million	
\
\
Year 2 year-end value after fees:	
\
\
119.0 – 3.37 = €115.63 million	
\
\
Year 2 after-fee return:	
\
\
$\Large\frac{115.63}{98.0}\small – 1 = 18.0\%$
